.env

```diff
@@ -20,3 +20,10 @@ ES_HOSTNAME=http://usaspending-es:9200
 
 # ES_CLUSTER_DIR should point to a path outside, or it will be mounted w/Docker
 ES_CLUSTER_DIR=../docker_es
+
+# Configuration values for a connection string to a Broker database
+#    Only necessary for some management commands
+BROKER_USER=root
+BROKER_PASSWORD=password
+BROKER_HOST=broker-db
+BROKER_PORT=5432
```

Comment:
Can we specify what environment this is for? Seems scary having plaintext Passwords here unless these are just dev or something.

Reply:
What should be done to make it less scary? The values are obviously for Broker and aren't pointing to an actual DB

Reply:
Are these placeholder values that a user is supposed to fill in? At first glance I would assume this went to _some_ database.

Reply:
Looking at the rest of this file, it is consistent. It just ticked off some alarms when I assumed it was in code.